baghdad-sciences-journal.csl

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" page-range-format="expanded" default-locale="en-GB">
+  <info>
+    <title>Baghdad Sciences Journal</title>
```

Comment:
Looks like the title should include "Science", not "Sciences". See http://www.csw-journal.org/.